api/sys_mounts.go

```diff
@@ -133,6 +133,7 @@ type MountInput struct {
 	Description string            `json:"description"`
 	Config      MountConfigInput  `json:"config"`
 	Local       bool              `json:"local"`
+	PluginName  string            `json:"plugin_name,omitempty"`
```

Comment:
We should add a comment indicating this is a deprecated field (same for fields below).

Reply:
Addressed in https://github.com/hashicorp/vault/pull/5825/commits/a88a9c4654eb2e34841622c1e76cd974bb6e8fb5